vllm/model_executor/custom_op.py

```diff
@@ -69,3 +76,61 @@ def dispatch_forward(self):
             return self.forward_xpu
         else:
             return self.forward_cuda
+
+    @staticmethod
+    def _get_enabled_ops() -> Tuple[bool, Set[str], Set[str]]:
+        """
+        Parse the VLLM_ENABLE_CUSTOM_OPS environment variable to determine
+         which custom ops are enabled. By default, custom ops are enabled
+         if VLLM_TORCH_COMPILE_LEVEL < CompilationLevel.INDUCTOR.
+         Specifying 'all' or 'none' will override this default.
+
+        :return: A tuple of (default_on, enabled_ops, disabled_ops)
+        """
+
+        # filter empty strings
+        env_ops = list(
+            filter(lambda x: len(x) > 0, envs.VLLM_ENABLE_CUSTOM_OPS))
+
+        use_all, use_none = env_ops.count("all"), env_ops.count("none")
+        assert use_all + use_none <= 1, \
+            "Cannot specify both 'all' and 'none' in VLLM_ENABLE_CUSTOM_OPS"
+
+        # On by default if VLLM_TORCH_COMPILE_LEVEL < CompilationLevel.INDUCTOR
+        default_on = envs.VLLM_TORCH_COMPILE_LEVEL < CompilationLevel.INDUCTOR
+
+        # override the default if 'all' or 'none' is specified
+        default_on = default_on and not bool(use_none) or bool(use_all)
+        enabled_ops, disabled_ops = set(), set()
+
+        for op in env_ops:
+            if op == "all" or op == "none":
+                continue
+
+            assert re.match(r"^-?[a-z0-9_]+$",
+                            op), f"Invalid custom op name: '{op}'"
+
+            if op.startswith("-"):
+                assert op[1:] not in {"all", "none"}, \
+                    f"Cannot disable all or none: '{op}'"
+                disabled_ops.add(op[1:])
+            else:
+                enabled_ops.add(op)
```

Comment:
this is too complicated. we can just use if `+rms_norm` in `envs.VLLM_CUSTOM_OPS`